opentracing-spring-cloud-starter-jaeger/README.md

```diff
@@ -0,0 +1,86 @@
+## Configuration options
+
+All the available configuration options can be seen in `io.opentracing.contrib.spring.cloud.starter.jaeger.JaegerConfigurationProperties`.
+The prefix to be used for these properties is `opentracing.jaeger`.
+
+Beware to use the correct syntax for properties that are camel-case in `JaegerConfigurationProperties`.
+
+* For properties / yaml files use `-`. For example `opentracing.jaeger.log-spans=true`
+* For environment variables use `_`. For example `OPENTRACING_JAEGER_LOG_SPANS` 
```

Comment:
Add a configuration section like in https://github.com/opentracing-contrib/java-spring-cloud#configuration and say that it is literaly everything what is needed :)

Reply:
Also add there how to configure operation name

Reply:
also say that by default it sends data to jaeger-agent via UDP + show how to change it and also how to use http sender

Reply:
:+1: